tests/fault_tolerance/deploy/container/build.sh

```
@@ -0,0 +1,183 @@
+
+set -e
+
+# Default values
+LOCAL_VLLM_IMAGE="vllm-elastic-ep:latest_all2all_buffer_input"
+DYNAMO_BASE_TAG="dynamo:latest-none"
+OUTPUT_TAG="my-dynamo-vllm:local"
+TARGET="dev"
+NO_CACHE=""
+BUILD_DYNAMO_BASE=true
+DOCKERFILE_DIR=$(dirname "$(readlink -f "$0")")
+PROJECT_ROOT=$(dirname $(dirname $(dirname $(dirname "$DOCKERFILE_DIR"))))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)